lib/generator/print_helpers.dart

```diff
@@ -113,8 +113,8 @@ Spec classDefinitionToSpec(ClassDefinition definition) {
       ..constructors.add(Constructor((b) {
         b
           ..optionalParameters.addAll(definition.properties
```

Comment:
those parameters on class constructors are actually only needed when we're generating input objects, but I'm fixing this behaviour on [visitor branch](https://github.com/comigor/artemis/tree/visitor)